src/SOS/SOS.NETCore/SymbolReader.cs

```diff
@@ -1106,43 +1116,47 @@ private static bool GetServerSymbolStore(ref SymbolStore store, bool msdl, bool
             {
                 // Use the internal symbol store for symweb
                 internalServer = symbolServerPath.Contains("symweb");
-
-                // Make sure the server Uri ends with "/"
-                symbolServerPath = symbolServerPath.TrimEnd('/') + '/';
             }
 
             if (symbolServerPath != null)
             {
                 // Validate symbol server path
-                if (!Uri.TryCreate(symbolServerPath, UriKind.Absolute, out Uri uri) || uri.IsFile)
+                if (!Uri.TryCreate(symbolServerPath.TrimEnd('/') + '/', UriKind.Absolute, out Uri uri))
```

Comment:
Does URI creation fail here if it's not ending in /? Is this true even for files? Before you used to have `|| uri.IsFile`. That being said, I don't know of a file cache.

Reply:
Yes, it seems that the URI doesn't have the trailing / on TryCreate and the SymStore code needs it. It would have been better if the HttpSymbolStore constructor/code made sure there was a trailing but it doesn't currently.

Reply:
The uri.IsFile moved to the line below with uri.IsUnc to add them as a "cache" path which assumes the name/{index}/name file structure.